slasher/server.go

```diff
@@ -0,0 +1,65 @@
+package slasher
```

Comment:
should this file be under `slasher/rpc/server.go`? For better clarity

Reply:
moved